modules/RTC/TPCUtils.js

```diff
@@ -80,32 +105,76 @@ export class TPCUtils {
-            ? this.encodingBitrates.ssHigh : this.encodingBitrates.high;
-
-        return [
+    _getVideoStreamEncodings(videoType, codec) {
+        const codecBitrates = this.codecSettings[codec].maxBitratesVideo;
+        const maxVideoBitrate = videoType === VideoType.DESKTOP
+            ? codecBitrates.ssHigh : codecBitrates.high;
+        const standardSimulcastEncodings = [
             {
                 active: this.pc.videoTransferActive,
-                maxBitrate: browser.isFirefox() ? maxVideoBitrate : this.encodingBitrates.low,
+                maxBitrate: browser.isFirefox() ? maxVideoBitrate : codecBitrates.low,
                 rid: SIM_LAYER_1_RID,
                 scaleResolutionDownBy: browser.isFirefox() ? HD_SCALE_FACTOR : LD_SCALE_FACTOR
             },
             {
                 active: this.pc.videoTransferActive,
-                maxBitrate: this.encodingBitrates.standard,
+                maxBitrate: codecBitrates.standard,
                 rid: SIM_LAYER_2_RID,
                 scaleResolutionDownBy: SD_SCALE_FACTOR
             },
             {
                 active: this.pc.videoTransferActive,
-                maxBitrate: browser.isFirefox() ? this.encodingBitrates.low : maxVideoBitrate,
+                maxBitrate: browser.isFirefox() ? codecBitrates.low : maxVideoBitrate,
                 rid: SIM_LAYER_3_RID,
                 scaleResolutionDownBy: browser.isFirefox() ? LD_SCALE_FACTOR : HD_SCALE_FACTOR
             }
         ];
+
+        if (this.codecSettings[codec].scalabilityModeEnabled) {
+            // Configure all 3 encodings for the simulcast case.
+            if (this.codecSettings[codec].useSimulcast || codec === CodecMimeType.H264) {
+                for (const encoding of standardSimulcastEncodings) {
+                    encoding.scalabilityMode = VideoEncoderScalabilityMode.L1T3;
+                }
+
+                return standardSimulcastEncodings;
+            }
+
+            // Configure only one encoding for the SVC mode.
+            return [
+                {
+                    active: this.pc.videoTransferActive,
+                    maxBitrate: maxVideoBitrate,
+                    rid: SIM_LAYER_1_RID,
+                    scaleResolutionDownBy: HD_SCALE_FACTOR,
+                    scalabilityMode: this.codecSettings[codec].useL3T3
+                        ? VideoEncoderScalabilityMode.L3T3 : VideoEncoderScalabilityMode.L3T3_KEY
+                },
+                {
+                    active: false
+                },
+                {
+                    active: false
+                }
+            ];
+        }
+
+        return standardSimulcastEncodings;
+    }
+
+    /**
+     * Returns a boolean indicating whether the video encoder is running in SVC mode.
+     *
+     * @param {CodecMimeType} codec
+     * @returns boolean
+     */
+    _isRunningInSvcMode(codec) {
+        return (codec === CodecMimeType.VP9 || codec === CodecMimeType.AV1)
```

Comment:
I am not sure I understand your question. 
VP8 only supports simulcast, i.e., 3 streams of different resolutions and each stream is temporally scalable. 
VP9 and AV1 support true SVC mode where one stream is both temporally and spatially scalable. When this returns true, the client makes sure that only the first encoding is correctly configured depending on what resolution is being requested keep the other 2 encodings disabled always.

Reply:
I mean to convey that a function called `_isRunningInSvcMode` might not be very clear since VP8 also does SVC, except only temporal, right?

How about `_isRunningInFullSvcMode` or `_isRunningInRealSvcMode` or `_isRunningInNonGmoSvcMode` ...

Reply:
Ok, got it. I will change it to `_isRunningInFullSvcMode`.